### DISCUSSION TITLE
webhook; access to tcp headers and sender ipaddress

### DISCUSSION THREAD

**Discussion by williamvm** [Budibase/budibase#7571] [Help]
Hello,
i created a budibase webhook post that receives a json body and can nicely save to a colum in new row in budibase db.
Is there a way to read tcp headers and senders ipaddress too? Looks like i only have access to the tcp body.
Somewhat similar to #2201. I thought Webhook is always incoming.
Thanks for your help!

**Comment by melohagan** [comment]
That's interesting, I didn't know the webhooks used to be that way - before my time!

I would be good to receive the headers of the incoming request - I was talking about this yesterday! https://github.com/Budibase/budibase/issues/7545#issuecomment-1233105838
